### ISSUE TITLE
PUT Request JSON Body Erroneously Being Sent as Query Parameters

### ISSUE THREAD

**Issue by JaredPotter** [usebruno/bruno#158]
When I create a PUT request, select the Body, set to JSON, and then put a valid JSON in the body edit box, and click "Send" - the data is sent as a query string.

E.g. 
Body:
```
{
    "version_identifier":"3.2.3",
    "major_version": "v3"
}
```

is read as `version_identifier=3.2.3&major_version=v3`.

In contrast - the exact same request in Postman is read correctly in our Node/Typescript API.

**Comment by helloanoop** [comment]
Hi @JaredPotter there was a bug that got introduced in v0.12.1
I have fixed it in v0.12.2 
Can you download the latest version and retry it ?

**Comment by helloanoop** [comment]
@JaredPotter closing this issue. let me know if this issue persists.
